README.md

```diff
@@ -228,12 +228,17 @@ v run tetris/tetris.v
 
 In order to build Tetris or 2048 (or anything else using `sokol` or `gg` graphics modules),
 you will need additional development libraries for your system.
-For some Linux distros (Debian/Ubuntu based), you need to run this:
-`sudo apt install libxi-dev libxcursor-dev`.
-For Fedora/RH/CentOS, you need to run this:
-`sudo dnf install libXcursor-devel libXi-devel libX11-devel libglvnd-devel` .
-For NixOS, add these packages to your `environment.systemPackages`:
-`xorg.libX11.dev xorg.libXcursor.dev xorg.libXi.dev libGL.dev`
+
+```bash
+Debian/Ubuntu based:
```

Comment:
That is a title/label and should be preferably outside the code block, or if you want it to be inside it, prepend a # comment sign for that line.

For me the point of the shell code blocks is that they indicate small snippets that can be directly copy/pasted and then run into a terminal. Putting the titles inside them, without a comment sign, goes against that purpose.

Reply:
I agree - it is more compact, and easier to read.